packages/service-library/src/servicelib/archiving_utils/_tdqm_utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Final
+
+from pydantic import NonNegativeFloat
+
+_UNIT_MULTIPLIER: Final[NonNegativeFloat] = 1024.0
+TQDM_FILE_OPTIONS: Final[dict[str, Any]] = {
+    "unit": "byte",
+    "unit_scale": True,
+    "unit_divisor": 1024,
+    "colour": "yellow",
+    "miniters": 1,
+}
+TQDM_MULTI_FILES_OPTIONS: Final[dict[str, Any]] = TQDM_FILE_OPTIONS | {
+    "unit": "file",
+    "unit_divisor": 1000,
+}
+
+
+def human_readable_size(size, decimal_places=3):
```

Comment:
this library has access to pydantic now right? so just use ByteSize instead. it has a human_readable function